mail_templates/assisted_living_checkin_html.njk

```diff
@@ -0,0 +1,25 @@
+<h1>{{ emailTitle }}</h1>
+
+<p>The Baltimore City Health Department’s Division on Aging is sending out a survey to licensed Assisted Living Facilities. This survey is optional, and will be used to understand resource needs for assisted living facilities in Baltimore City. If you choose to complete this survey, all questions will need to be answered to better assist The Baltimore City Health Department in assessing areas of need or concern. All responses will be kept confidential. </p> 
+
+<p>The Baltimore City Health Department will assess needs and work to help with resources to the best of our ability. This survey will not be used to give out supplies, but may be used to help find resources on where facilities can access those supplies if there is a shortage.</p>
+
+<p>We also want to provide you with additional information about support that is available to licensed facilities during COVID-19. Please join our weekly calls on Fridays from 2pm-3pm for further information on how to best protect residents and staff during this time. Information from these surveys will be used to understand topics that need to be covered in this weekly call. A link to the online meeting will be emailed out separately. </p>
```

Comment:
Please change to "2:00 PM - 3:00 PM ET"

Reply:
The text was sent by the city. Don't want to change their copy.